_rules/aria-presentational-role-not-focusable-18pg11.md

```diff
@@ -0,0 +1,139 @@
+---
+id: 18pg11
+name: ARIA presentational role not focusable
```

Comment:
See above, the [Element marked as decorative is not exposed](https://www.w3.org/WAI/standards-guidelines/act/rules/46ca7f/#marked-as-decorative) rule covers this.

I think we can debate splitting the rule up into the two rules you propose. I'd personally lean towards leaving this as is, since the "not exposed" rule is already approved by AG, but if others feel there are good reasons to deprecate it in favor of this I'm open to that conversation.

Reply:
I see the point. 

The intent was to cover a specific ARIA requirement in detail and comprehensively (Presentational Roles Conflict Resolution) to guarantee authors are clear on what to do in these scenarios. 

I'm uncertain about the next steps. 

The other two existing ARIA requirements are rather more vague but do cover the necessary aspects. If we decide to revert these new rules, we'll still need to add examples to the existing rules.